.github/workflows/run-network-tests.yml

```diff
@@ -104,8 +104,15 @@ jobs:
           yarn workspace @joystream/cli build
       - name: Ensure tests are runnable
         run: yarn workspace network-tests build
+      - name: Install joystream-cli @joystream/cli/0.5.1
+        run: npm -g install @joystream/cli
```

Comment:
Perhaps we should lock the version to `@joystream/cli@^0.5.1` just in case

Reply:
I add that in next PR :+1: